### DISCUSSION TITLE
네이버 본인인증이 안됩니다

### DISCUSSION THREAD

**Discussion by seokhyeong9171** [changdoc/naver-vaccine-macro#158] [Q&A]
아무리 인증서 본인인증을 해도 본인인증탭에서 예약신청 탭으로 안넘어가지네요
크롬 시크릿모드에서는 넘어가지는데 시크릿모드로 해도 아무런 문제 없을까요?

**Comment by changdoc** [comment]
시크릿 모드에서 동작 시킨 사례는 들어본적이 없긴한데 별 문제는 없을 것 같습니다.

성공하시면 공유해주시면 감사하겠습니다~
